packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -353,6 +354,13 @@ export class LedgerHDWallet implements core.HDWallet, core.BTCWallet, core.ETHWa
     }
   }
 
+  public async listApps (): Promise<any> {
```

Comment:
Is it possible to be more specific with the return type? i.e. `Promise<string[]>` or something?